### PR TITLE
Custom dropdown

### PR DESCRIPTION
**Pull request by zefwang**
2 months in the works, and it's finally ready. Sorta?

### CODE REVIEW

**Review by NEUDitao — APPROVED** [review]
i scroleld down and looked at the changes

**Review by beccajohnson17 — COMMENTED** [review]
Is it possible you could add screenshots? :)

**Comment by zefwang** [comment]
Dropdown normally
Screen Shot 2020-07-13 at 9 26 25 PM

Dropped down
Screen Shot 2020-07-13 at 9 26 34 PM

Selected, one row
Screen Shot 2020-07-13 at 9 26 49 PM

Selected, multiple rows
Screen Shot 2020-07-13 at 9 26 59 PM

@beccajohnson17 feel free to pull branch and see if there are any behaviors you don't like!

File: frontend/components/ResultsPage/DropdownFilter.tsx [hunk 1]
```diff
@@ -11,25 +15,70 @@ interface DropdownFilter {
 export default function DropdownFilter({
   title, options, selected, setActive,
 }: DropdownFilter) {
+  const [areOptionsFresh, setAreOptionsFresh] = useState(true)
```
Comment (dajinchu) [inline frontend/components/ResultsPage/DropdownFilter.tsx]: we can get rid of this since it sounds like the performance has not been an issue

**Review by dajinchu — APPROVED** [review]
👍🏼 just get rid of areOptionsFresh

**Comment by zefwang** [comment]
Screen Shot 2020-08-13 at 12 25 40 PM
@beccajohnson17

**Pull request merged**